source/frontends/libretro/retroregistry.cpp

```diff
@@ -171,4 +181,29 @@ namespace ra2
     return registry;
   }
 
+  u_int32_t lookupValue(std::string name, std::string key)
```

Comment:
const &

Reply:
It is not a pair in the method signature, I don't see how const & ... would work here.